manual_hook.rb

```diff
@@ -20,7 +18,14 @@ def setup_dns(domain, txt_challenge)
        puts "Found #{resp.strings[0]}. no match."
      end
     }
+
     if !resolved
+
+     puts "Create TXT record for the domain: \"#{acme_domain}\". TXT record:"
```

Comment:
Maybe wrap this whole section in an if statement so it only gets executed once? That way there are no readlines once we've `in theory` updated our txt record, and the tool can just loop every 30 seconds with no interaction